ToDoList-Redux/Binders/TaskListViewModel.swift

```diff
@@ -11,24 +11,17 @@ import SwiftRex
 import CombineRex
 
 extension TaskList {
-    
     // MARK: - STATE
     struct State: Equatable {
         var title: String
-        var tasks: [Task]
+        var tasks: [Item]
```

Comment:
Before I mentioned that AppState tree should probably not have View State.
The opposite is not necessarily true, you could use Task (model) here. But I usually recommend using custom view state so you can do date formatting, localization, "flattenize" your model into a struct that corresponds to your view hierarchy and distribution. Besides, this helps reducing SwiftUI updates.